### PR TITLE
Upgrade to VS 2017

### PR DESCRIPTION
**Pull request by natemcmaster**
Remove project.json and upgrade to MSBuild.

Also, rev the package version to 0.3.0. It was building as 0.2.0-preview1, but we already shipped "0.2.0".

cref https://github.com/aspnet/Universe/issues/469

cc @mikaelm12

### CODE REVIEW

File: src/Microsoft.AspNetCore.Proxy/Microsoft.AspNetCore.Proxy.csproj [hunk 1]
```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="..\..\build\common.props" />
+
+  <PropertyGroup>
+    <Description>Proxy Library</Description>
+    <VersionPrefix></VersionPrefix>
```
Comment (BrennanConroy) [inline src/Microsoft.AspNetCore.Proxy/Microsoft.AspNetCore.Proxy.csproj]: 0.3.0?

Comment (natemcmaster) [inline src/Microsoft.AspNetCore.Proxy/Microsoft.AspNetCore.Proxy.csproj]: Darn the luck. Code-gen strikes again 😈

**Comment by BrennanConroy** [comment]
:ship: :it:

**Comment by mikaelm12** [comment]
Yay Proxy 🎉

**Pull request merged**